neo/rawio/tdtrawio.py

```diff
@@ -248,14 +249,21 @@ def _parse_header(self):
                         sev_filename = (path / sev_stem).with_suffix('.sev')
                     else:
                         # for single block datasets the exact name of sev files in not known
-                        sev_regex = f".*_ch{chan_id}.sev"
+                        sev_regex = f"*_Ch{chan_id}.sev"
```

Comment:
I think it would be good to also preserve the old matching behaviour here:
```suggestion
                        sev_regex = f"*_[cC]h{chan_id}.sev"
```

Reply:
This seems to work fine with my files